L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -32,4 +34,11 @@
                                                TrackQuality = cms.bool(False),
                                                TrackQualityPSet = cms.PSet(TrackQualityParams)
     )
+TTTracksFromReducedTrackletEmulation = TTTracksFromTrackletEmulation.clone(
```

Comment:
Please add comment saying what reduced chain actually is. Otherwise, it's unclear if this will only be useful with the current summer chain, or may also be useful for other reduced chains in future.